### DISCUSSION TITLE
single activation for cookies/cat

### DISCUSSION THREAD

**Discussion by yannikbiz** [orestbida/cookieconsent#406] [Q&A]
Can i get a exampe of this image 

![image](https://user-images.githubusercontent.com/117126354/199329944-721db612-5588-40ea-b885-b6a00e1dde46.png)

**Comment by orestbida** [comment]
Hi @yannikbiz,

I don't have access to the files right now, but you can find examples on how to configure individually togglable services in the [docs.](https://cookieconsent.orestbida.com/).

Here are some useful resources:
* [How to manage script tags](https://cookieconsent.orestbida.com/advanced/manage-scripts.html#how-to-manage-scripts)
* [How to define a service internally](https://cookieconsent.orestbida.com/reference/configuration-reference.html#category-services)
* [How to configure cookie autoClear](https://cookieconsent.orestbida.com/reference/configuration-reference.html#category-autoclear)

You can also check a [full example with GTM here](https://github.com/orestbida/cookieconsent/tree/v3.0-beta/demo/demo_gtm).

If you still have trouble configuring the plugin, I'll try to send my own config. whenever I get to my main pc.

**Reply by yannikbiz** [reply]
thansk for fast answer !!!!

**Reply by edgarhuber** [reply]
@yannikbiz If you are looking for a way to integrate the services and badges you should check out the iframemanager demo: 

https://github.com/orestbida/cookieconsent/tree/v3.0-beta/demo/demo_iframemanager

That's what helped me a lot.
